ios/Exponent/Kernel/AppLoader/CachedResource/EXManifestResource.m

```diff
@@ -406,17 +406,17 @@ - (NSError *)formatError:(NSError *)error
   if ([errorCode isEqualToString:@"EXPERIENCE_NOT_FOUND"]
       || [errorCode isEqualToString:@"EXPERIENCE_NOT_PUBLISHED_ERROR"]
       || [errorCode isEqualToString:@"EXPERIENCE_RELEASE_NOT_FOUND_ERROR"]) {
-    formattedMessage = [NSString stringWithFormat:@"No experience found at %@.", self.originalUrl];
+    formattedMessage = [NSString stringWithFormat:@"No project found at %@.", self.originalUrl];
```

Comment:
so nice to be moving away from "experience"

Reply:
yep, this will almost certainly be more intuitive to our developer users and more closely match the modern way that people use expo